include/LocalHostStats.h

```diff
@@ -46,7 +46,7 @@ class LocalHostStats: public HostStats {
 
   /* Estimate the number of contacted countries using HyperLogLog */
   struct ndpi_hll hll_countries_contacts;
-  double old_hll_countries_value, new_hll_countries_value, hll_delta_countries_value;
+  u_int8_t old_hll_countries_value, new_hll_countries_value, hll_delta_countries_value;
```

Comment:
@simonemainardi actually we fixed it. Is it possible you were reading line 44 instead of 49? Line 44 isn't part of our code.